carp.studies.core/src/commonMain/kotlin/dk/cachet/carp/studies/domain/Recruitment.kt

```diff
@@ -0,0 +1,109 @@
+package dk.cachet.carp.studies.domain
+
+import dk.cachet.carp.common.UUID
+import dk.cachet.carp.common.ddd.AggregateRoot
+import dk.cachet.carp.common.ddd.DomainEvent
+import dk.cachet.carp.deployment.domain.users.Participation
+import dk.cachet.carp.deployment.domain.users.StudyInvitation
+import dk.cachet.carp.protocols.domain.StudyProtocolSnapshot
+import dk.cachet.carp.studies.domain.users.DeanonymizedParticipation
+import dk.cachet.carp.studies.domain.users.Participant
+
+
+/**
+ * Represents a set of participants recruited for a [Study] identified by [studyId].
+ */
+class Recruitment( val studyId: UUID ) :
```

Comment:
Do we want to also store `Participant`s, or at least their identities, in `Recruitment`?